ocs_ci/deployment/deployment.py

```diff
@@ -3451,7 +3451,11 @@ def build_bucket_name(self, acm_indexes):
     @retry((TimeoutExpiredError, ACMClusterConfigurationException), tries=20, delay=10)
     def backup_pod_status_check(self):
         pods_list = get_all_pods(namespace=constants.ACM_HUB_BACKUP_NAMESPACE)
-        if len(pods_list) != 3:
+        backup_pod_count = 2
+        ocp_version = version.get_semantic_ocp_version_from_config()
+        if ocp_version < version.VERSION_4_19:
+            backup_pod_count = 3
+        if len(pods_list) != backup_pod_count:
```

Comment:
we need to have 2 conditions 

- 4.19 OCP

-  2.14 ACM

We cannot directly set backup_pod_count to 2 if 4.19 . we need to make sure if pod count is changed with acm 2,13 with ocp 4.19